src/AppInstallerCommonCore/Downloader.cpp

```diff
@@ -128,11 +128,25 @@ namespace AppInstaller::Utility
         return result;
     }
 
+    std::optional<std::vector<BYTE>> DownloadToStream(

```

Comment:
>DownloadToStream [](http://example.com/codeflow?start=37&length=16)

Do we expect to make this use DO in the future, currently it's only used by manifest downloading and this extra level of wrapping seems unnecessary.

Reply:
Maybe; I had originally implemented the split here, but I did not understand how to make DO do an in memory download of the full file.  I do now, but I don't see a reason to change back here.  I do want to have the type continue to come in to the download though, so we can properly categorize them.